tests/sqlSyncTest.php

```diff
@@ -44,12 +44,13 @@ public function localSqlSync() {
       'yes' => NULL,
     );
     $this->drush('user-create', array($name), $options + array('password' => 'password', 'mail' => $mail));
+    $this->drush('pm-enable', array('comment'), $options + array('yes' => NULL));
```

Comment:
Ah, thanks for clarifying that we are only talking about fields on user entity. There is no code to copy in this repo for adding Fields to users. However, I found [this code in Drupal that you can copy](https://api.drupal.org/api/drupal/core%21modules%21field%21tests%21src%21Kernel%21Boolean%21BooleanItemTest.php/function/BooleanItemTest%3A%3AsetUp/8.2.x)